src/inspector_agent.cc

```diff
@@ -387,29 +408,29 @@ bool Agent::Start(v8::Platform* platform, const char* path,
           new NodeInspectorClient(parent_env_, platform));
   inspector_->contextCreated(parent_env_->context(), "Node.js Main Context");
   platform_ = platform;
+  CHECK_EQ(0, uv_async_init(uv_default_loop(),
+                            &start_inspector_thread_async,
+                            StartInspectorIoThreadAsyncCallback));
+  start_inspector_thread_async.data = this;
+  uv_unref(reinterpret_cast<uv_handle_t*>(&start_inspector_thread_async));
+
+  RegisterDebugSignalHandler();
```

Comment:
You are right. Signal handler was made unconditional to support restarting the WS server after process._debugEnd()